test/verify/check-packagekit

```diff
@@ -1898,12 +1898,30 @@ class TestInstallDialog(NoSubManCase):
         b.wait_text("#dialog .scale-up-ct", "Additional packages:test-webengine")
         self.install_packages()
 
-        # Test conflicting packages are shown as removed
-        self.show_install_dialog("breaking-software")
-        b.wait_in_text("#dialog", "breaking-software will be installed.")
-        # The svg icon creates a tiny bit of whitespace
-        b.wait_text("#dialog .scale-up-ct", " Removals:ok-software")
-        self.install_packages()
+        if self.backend != "dnf5":
+            # Test conflicting packages are shown as removed
+            self.show_install_dialog("breaking-software")
+            b.wait_in_text("#dialog", "breaking-software will be installed.")
+            # The svg icon creates a tiny bit of whitespace
+            b.wait_text("#dialog .scale-up-ct", " Removals:ok-software")
+            self.install_packages()
+        else:
+            # package which is replaced with another package
+            self.createPackage("good-software", "998", "1", install=True)
+            self.createPackage("awesome-software", "999", "1", provides="good-software = 999", obsoletes="good-software < 999")
+            self.enableRepo()
+
+            self.show_install_dialog("awesome-software")
+            b.wait_in_text("#dialog", "awesome-software will be installed.")
+            # The svg icon creates a tiny bit of whitespace
+            b.wait_text("#dialog .scale-up-ct", " Removals:good-software")
+            self.install_packages()
+
+            # Conflicting packages require setting `allow_erasing`.
+            self.show_install_dialog("breaking-software")
+            b.wait_in_text("#dialog", "breaking-software will be installed.")
+            b.wait_in_text("#dialog .pf-v6-c-alert__title", "Resolving install failed with result=2")
```

Comment:
eek, do we really show the numeric error code in the UI? I thought everything used `get_transaction_problems_string()` now, could it check that and avoid the number?

Reply:
There is also a string, see 

<img width="821" height="282" alt="image" src="https://github.com/user-attachments/assets/8a7cd41f-f4f5-4481-8971-34acaf9712c5" />

Reply:
Ack, good 'nuff!